apps/taiga-lumbermill/src/dashboards/iot/components/music/music.component.html

```diff
@@ -0,0 +1,109 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+    class="card"
+>
+    <header tuiHeader>
+        <h2
+            tuiTitle
+            [style.text-align]="'center'"
+        >
+            Music
+        </h2>
+    </header>
+    <div class="info">
+        <img
+            class="logo"
+            [src]="musicService[curMusic].logo"
+        />
+        <h2 class="tui-space_top-4 tui-space_bottom-2">
+            {{ musicService[curMusic].name }}
+        </h2>
+        <p
+            class="tui-space_top-0 tui-space_bottom-0"
+            [style.color]="'var(--tui-text-primary)'"
+        >
+            {{ musicService[curMusic].performer }}
+        </p>
+    </div>
+    <audio
+        #audio
+        tuiMedia
+        [src]="musicService[curMusic].src"
+        [(currentTime)]="curTime"
+        [(paused)]="paused"
+        [(volume)]="volume"
```

Comment:
try to start playing when "next" clicked or previous track ended ( `(ended)` event may be helpful)

Reply:
I added event (ended), but if I set the track to start immediately in the previousTrack and nextTrack functions, then nothing happens